builtin/aws/ecs/task.go

```diff
@@ -63,10 +63,10 @@ type TaskLauncherConfig struct {
 
 	// Subnets are the list of subnets for the cluster. These will match the
 	// subnets used for the Cluster
-	Subnets string `hcl:"subnets,required"`
+	Subnets string `hcl:"subnets"`
 
 	// SecurityGroupId is the security group used for the Waypoint tasks.
-	SecurityGroupId string `hcl:"security_group_id,required"`
+	SecurityGroupId string `hcl:"security_group_id"`
```

Comment:
Hm, based on the PR description, should these explicitly be `optional`?

> required is not a valid hcl tag. In hcl, it looks like the absence of an "optional" tag indicates that a field is required.

Reply:
@briancain though `required` is not the correct HCL tag (introduced by me), the fields should indeed be listed as required, because ODRs would not launch without the fields being set (#3514).

Reply:
Got it! I understand now